css/index.css

```
@@ -0,0 +1,535 @@
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    width: 15.9rem;
+    height: 1.9rem;
+    gap: 3rem;    
+}
+.footer-privacy-link {
+    width: 10rem;
+    height: 1.9rem;
+    font-weight: 400;
+    font-size: 1.6rem;
+    line-height: 100%;
+    text-decoration: none;
+    color: var(--secondary-200);
+}
+.footer-faq-link {
+    width: 2.9rem;
+    height: 1.9rem;
+    font-weight: 400;
+    font-size: 1.6rem;
+    line-height: 100%;
+    text-decoration: none;
+    color: var(--secondary-200);
+}
+.footer-sns {
+    display: flex;
+    justify-content: space-between;
+    width: 11.6rem;
+    height: 2rem;
+    gap: 1.2rem;
+    margin: 0;
+    padding: 0;
+    flex-wrap: nowrap;
+}
+.sns-item {
+    width: 2rem;
+    height: 2rem;
+    list-style-type: none;
+} 
+
+@media  (min-width: 768px) and (max-width: 1199px){
+    html {
+        font-size: 9px;
+    }
+    header {
+        padding: 0 2.4rem;
+    }
+    .banner-container {
+        height: 77.1rem;
+        flex-direction: column;
+        justify-content: flex-end;
+        align-items: center;
+    }
+    .banner-content {
+        width: 100%;
+        height: 69.7rem;
+        flex-direction: column;
+    }
+    .banner-text {
+        width: 100%;
+        height: 13.6rem;
+        gap: 2.4rem;
+        flex-direction: column;
+        align-items: center;
+        justify-content: space-between;
+    }
+    .banner-message, .banner-message-enter {
+        text-align: center;
+        width: 100%;
+        height: 5.6rem;
+        display: inline;
+    }
+    .panda-image {
+        width: 74.4rem;
+    }
+    .section-main {
+        padding: 2.4rem;
+        height: 70.8rem;
+        gap: 5.2rem;
+    }
+    .section-content {
+        width: 69.6rem;
+        height: 70.8rem;
+        gap: 2.4rem;
+        flex-direction: column;
+        align-items: flex-start;
+    }
+    .section-image {
+        width: 69.6rem;
+        height: 52.4rem;
+    }
+    .section-text {
+        width: 31.9rem;
+        height: 16rem;
+        gap: 1.6rem;
+        flex-wrap: nowrap;
+    }
+    .card-title {
+        height: 2.6rem;
+        width: 100%;
+    }
+    .text-card-info {
+        width: 31.9rem;
+        height: 11.8rem;
+        gap: 0;
+    }
+    .section-content-right {
+        display: flex;
+        flex-direction: column-reverse;
+        align-items: flex-end;
+    }
+    .section-right {
+        display: flex;
+        text-align: right;
+        align-items: flex-end;
+    }
+    .text-card-right {
+        display: flex;
+        align-items: flex-end
+    }
+    .best-item-title, .item-text {
+        height: 4.2rem;
+        width: 100%;
+        display: inline;
+        font-size: 2.4rem;
+    }
+    .best-item-description {
+        height: 5.2rem;
+        font-size: 1.8rem;
+        line-height: 2.6rem;
+    }
+    footer {
+        padding: 3.2rem 10.2rem 0;
+    }
+}
+@media (min-width: 375px) and (max-width: 767px){
```

❗️ 수정요청
min-width 조건이 들어가게 되면 374px 부터는 해당 스타일이 적용되지 않으니 아래처럼 max-width만 남겨두셔서 최대한 반응형 디자인을 지원하시는 것을 추천드려요~

```suggestion
@media (max-width: 767px){
```